deploy/chart/templates/controller.yaml

```diff
@@ -315,6 +315,10 @@ spec:
             - name: ECS_ENDPOINT
               value: {{ .Values.deploy.ecsEndpoint | quote }}
 {{- end }}
+{{- if .Values.deploy.stsEndpoint }}
+            - name: STS_ENDPOINT
```

Comment:
STS_ENDPOINT should be renamed to RRSA_STS_ENDPOINT, as it's currently only used in RRSA

Reply:
But if we invoke other APIs of STS in the future, we should still use this endpoint.
BTW, RRSA is a concept for ACK, it should be OIDC for STS.

Reply:
We should modify the endpoint in metadata as well to match stsclient in the metadata openapi fetcher

Reply:
To utilize the universal STS_ENDPOINT  env name.

Reply:
I submit a new commit to set endpoint when try new a stsclient.